src/main/java/org/folio/config/RMAPIConfiguration.java

```diff
@@ -25,22 +31,6 @@ public Boolean getConfigValid() {
     return configValid;
   }
 
-  public void setCustomerId(String customerId) {
-    this.customerId = customerId;
-  }
-
-  public void setApiKey(String apiKey) {
-    this.apiKey = apiKey;
-  }
-
-  public void setUrl(String url) {
-    this.url = url;
-  }
-
-  public void setConfigValid(Boolean configValid) {
-    this.configValid = configValid;
-  }
-
   @Override
   public String toString() {
```

Comment:
you can potentially remove toString(), it can be generated by Lombook for @Value classes

Reply:
Thanks, done